test/python/compiler/test_transpiler.py

```diff
@@ -1447,6 +1454,80 @@ def test_scheduling_backend_v2(self):
         self.assertIn("delay", out[0].count_ops())
         self.assertIn("delay", out[1].count_ops())
 
+    def test_scheduling_timing_constraints(self):
+        """Test that scheduling-related loose transpile constraints
+        work with both BackendV1 and BackendV2."""
+
+        backend_v1 = Fake27QPulseV1()
+        backend_v2 = BackendV2Converter(backend_v1)
+        # the original timing constraints are granularity = min_length = 16
+        timing_constraints = TimingConstraints(granularity=32, min_length=64)
+        error_msgs = {
```

Comment:
I don't think that the timing constraints are encoded anywhere in the output, so the only strategy I could think of for testing was triggering error messages that would expose the constraints.